lms/djangoapps/course_goals/management/commands/goal_reminder_email.py

```diff
@@ -51,29 +51,38 @@ def send_ace_message(goal):
     site = Site.objects.get_current()
     message_context = get_base_template_context(site)
 
-    course_home_url = get_learning_mfe_home_url(course_key=goal.course_key)
+    course_home_url = get_learning_mfe_home_url(course_key=goal.course_key, view_name='home')
 
     goals_unsubscribe_url = settings.LEARNING_MICROFRONTEND_URL + reverse(
         'course-home:unsubscribe-from-course-goal',
         kwargs={'token': goal.unsubscribe_token}
     )
 
+    image_url = settings.LMS_ROOT_URL + settings.STATIC_URL + 'images/'
+    language = get_user_preference(user, LANGUAGE_KEY)
+
+    # Code to allow displaying different banner images for different languages
+    # However, we'll likely want to develop a better way to do this within edx-ace
```

Comment:
What do you mean by this? This part of the comment isn't clear to me.

Reply:
i was going to file a ticket to come up with some more extensible way of using different assets for different languages
for instance maybe we make it so you put the assets in a folder labeled with the language code or something, not sure

Reply:
Ah sure. But wouldn't that be an edx-platform concern anyway? Or are you also proposing moving these images into edx-ace?